group_vars/development

```diff
@@ -30,6 +32,24 @@ wordpress_sites:
       db_user: example_dbuser
       db_password: example_dbpassword
 
+mail_smtp_server: localhost:1025
+mail_admin: admin@example.dev
+mail_hostname: example.dev
+ssmtp_auth_method: ""
+ssmtp_start_tls: 'no'
+ssmtp_tls: 'no'
+
+ferm_input_list:
+  - type: dport_accept
+    dport: [http, https]
+    filename: nginx_accept
+  - type: dport_accept
+    dport: [8025]
+  - type: dport_limit
+    dport: [ssh]
+    seconds: 300
+    hits: 20
```

Comment:
This block is mostly a copy of the one in the `all` settings. Because it is not possible to append to a list in Ansible its not possible to only add the rule to open port `8025`, I have to override the whole list.

another option would be to put `ferm_enabled: false` for development by default. what do you guys think?